tests/modules/cnvkit/batch/main.nf

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CNVKIT as CNVKIT_TUMORONLY } from '../../../../modules/cnvkit/batch/main.nf' addParams( options: [ 'args': '--reference reference.cnn --method wgs' ] )
+include { CNVKIT }               from '../../../../modules/cnvkit/batch/main.nf' addParams( options: [ 'args': '--output-reference reference.cnn' ] )
```

Comment:
can you align this on `from` and `addParams`

Reply:
👍